README.md

```diff
@@ -75,3 +73,27 @@ This command only works if:
 
 - you are in a local Git repository
 - the repository has a file called ‘.pre-commit-config.yaml’
+
+``` bash
+spaid_create_function_file -h
+```
+
+Usage: spaid_create_function_file \[-h\]
+
+Run this script to create a new function file with the corresponding
+test file.
+
+Example:
+
+    $ spaid_create_function_file seedcase_sprout core properties
```

Comment:
With the current folder structure in the `seedcase-sprout` repo, this should be:
```suggestion
    $ spaid_create_function_file sprout core properties
```
But maybe this is intentional as we have agreed to rename the `sprout` folder?

Reply:
I did that intentionally since we will be renaming the folder.